pkg/opendsme/contrib/Makefile

```diff
@@ -0,0 +1,3 @@
+MODULE = opendsme-contrib
```

Comment:
I see that we may want to distinguish this from other opendsme modules (because it's not representing a part of OpenDSME but rather what "we" tack on top), but I don't think that having one module with a dash in a series of underscore delimited modules is the pretty way to do it ... maybe `opendsme_riot_contrib` or more concretely `opendsme_gnrc`?

Reply:
I see. In that case `opendsme_riot_contrib` seems to be a good candidate, because in the future we might add other stacks as well (LWIP).

Reply:
done!

Reply:
I don't see this as resolved, latest branch head still has the `-contrib` name.

Reply:
uh, sorry, I forgot to push. It should be solved now